frontend/src/css/modules/landing.scss

```diff
@@ -110,8 +110,47 @@ section.bg-section {
     font-size: 18px;
     top: 0;
     left: 0;
-    z-index: 1001;
+    z-index: 999;
     width: 100%;
+    position: relative;
+    nav {
+        background-color:rgba(255, 255, 255, 0.9);
+        box-shadow: 0 0 0 0 #000;
+        position: fixed;
+        top: 0;
+        left: 0;
+        padding: 0 100px;
+        transition: all 0.1s ease-in-out;
+        a.sim-header-link {
+	  color: #4d4d4d;
+	  font-weight: 400;
+	  transition: all 0.2s ease-in-out;
+	}
+        a.sim-header-link:hover {
```

Comment:
@Obtin you can make these nested too using &.
Like a{&.sim-header-link{}}

Reply:
@aka-jain done.